polygeist/include/mlir/Dialect/Polygeist/IR/PolygeistTypes.td

```diff
@@ -25,4 +25,15 @@ class Polygeist_Type<string name, string typeMnemonic, list<Trait> traits = []>
   let mnemonic = typeMnemonic;
 }
 
+def Polygeist_StructType :
+    Polygeist_Type<"Struct", "struct"> {
+  let summary = "Polygeist Struct";
+  let description = [{
+    Structure type that allowed to be the element type of MemRef and LLVM
```

Comment:
[typo]: Structure type that *is* allowed ...

Reply:
done.